up42/utils.py

```diff
@@ -181,6 +183,24 @@ def folium_base_map(
     return m
 
 
+def _style_function(feature):  # pylint: disable=unused-argument
+    return {
+        "fillColor": "#5288c4",
+        "color": "blue",
+        "weight": 2.5,
+        "dashArray": "5, 5",
+    }
+
+
+def _highlight_function(feature):  # pylint: disable=unused-argument
+    return {
+        "fillColor": "#ffaf00",
+        "color": "red",
+        "weight": 3.5,
+        "dashArray": "5, 5",
+    }
+
+
```

Comment:
Can we try to get rid of the pylint disable things?

Reply:
Will try